src/controllers/logs.controller.ts

```diff
@@ -0,0 +1,28 @@
+import { Request, Response } from "express";
+import Log from "../models/Log";
+
+export const LogsController = {
+  async getAll(req: Request, res: Response) {
+    const logs = await Log.findAll({ order: [["createdAt", "DESC"]] });
+    res.json(logs);
+  },
+
+  async getLogsByBoard(req: Request, res: Response) {
+    const { boardId } = req.params;
+
+    const logs = await Log.findAll({
+      where: { boardId },
+      order: [["createdAt", "DESC"]],
+    });
+
+    const formattedLogs = logs.map((log) => ({
```

Comment:
What happened here? If you need to select just some fields, use `attributes`. There's not transformation going on in your mapping loop.